webui/master/src/constants/data/routePaths.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+export const routePaths = {
+    root: '/',
+    overview: '/overview',
+    browse: '/browse',
+    config: '/config',
+    data: '/data',
```

Comment:
Unrelated to this PR - we should probably change this path to be more descriptive.

Reply:
noted